core/test/martian/schema_test.cljc

```diff
@@ -394,3 +395,65 @@
 
     (is (= (s/maybe {(s/optional-key :b) (s/maybe {(s/optional-key :a) s/Any})})
            schema))))
+
+
+(def definition {:org.chaos-mesh.v1alpha1.NetworkChaos {:description "NetworkChaos is the Schema for the networkchaos API",
```

Comment:
I could use a json like swagger-issue-111.json to improve readability if you prefer